src/main/java/util/DataConverterImpl.java

```diff
@@ -0,0 +1,26 @@
+package util;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> lines) {
```

Comment:
The method `convertToTransaction` does not handle the case where the input lines may not be correctly formatted. It's important to include error handling for parsing the operation and quantity to ensure that invalid input data does not cause the program to crash. Consider adding a try-catch block around the parsing logic and throw a meaningful exception if parsing fails.